shared/templates/sudo_defaults_option/bash.template

```diff
@@ -11,15 +11,15 @@
 {{% endif %}}
 if /usr/sbin/visudo -qcf /etc/sudoers; then
     cp /etc/sudoers /etc/sudoers.bak
-    if ! grep -P "^[\s]*Defaults.*\b{{{ OPTION_REGEX }}}\b.*$" /etc/sudoers; then
+    if ! grep -P '^[\s]*Defaults.*\b{{{ OPTION_REGEX }}}\b.*$' /etc/sudoers; then
         # sudoers file doesn't define Option {{{ OPTION }}}
         echo "Defaults {{{ OPTION_VALUE }}}" >> /etc/sudoers
     {{%- if not VARIABLE_NAME %}}
     fi
     {{% else %}}
     else
         # sudoers file defines Option {{{ OPTION }}}, remediate if appropriate value is not set
-        if ! grep -P "^[\s]*Defaults.*\b{{{ OPTION_VALUE }}}\b.*$" /etc/sudoers; then
+        if ! grep -P '^[\s]*Defaults.*\b{{{ OPTION_VALUE }}}\b.*$' /etc/sudoers; then
```

Comment:
This is the line I'd leave alone; `OPTION_VALUE` can contain shell variables, per my earlier comment. :D

Reply:
Yes, that makes sense. Reverting it. Thanks.